<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>88</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Graph</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnShowGraph">
       <property name="text">
        <string>Show Graph</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCloseGraph">
       <property name="text">
        <string>Close Graph</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Graph settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textFieldStartNode">
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="placeholderText">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textFieldGoalNode">
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="placeholderText">
        <string>Goal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtFieldLimit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>depth (limit/iterative) </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>321</width>
      <height>99</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Algorithm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkboxDfs">
       <property name="text">
        <string>DFs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkboxLdfs">
       <property name="text">
        <string>Limited dfs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkboxIds">
       <property name="text">
        <string>Iterative deepening search</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>361</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="containerResults">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Results</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="txtExpandedNodes">
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="txtFinalPath">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>540</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Developed by Mohammad Nawas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>560</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2023</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>359</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
